tech.json

```diff
@@ -13,7 +13,10 @@
         {
           "name": "canPrepareForNextRoom",
           "requires": [],
-          "note": "The ability to perform an action as part of a strat that occurs following a room transition."
+          "note": [
+            "The ability to know what room is coming next and perform an action immediately on entry.",
```

Comment:
The action begins before entry.  Its taken as part of entering the room, I'd say.